build-tools/Xamarin.Android.Tools.BootstrapTasks/Properties/AssemblyInfo.cs

```diff
@@ -4,15 +4,10 @@
 // Information about this assembly is defined by the following attributes. 
 // Change them to the values specific to your project.
 
-[assembly: AssemblyTitle ("Xamarin.Android.Tools.BootstrapTasks")]
 [assembly: AssemblyDescription ("")]
-[assembly: AssemblyConfiguration ("")]
-[assembly: AssemblyCompany ("Xamarin Inc.")]
-[assembly: AssemblyProduct ("")]
 [assembly: AssemblyCopyright ("Xamarin Inc.")]
 [assembly: AssemblyTrademark ("Xamarin")]
 [assembly: AssemblyCulture ("")]
-[assembly: AssemblyVersion ("1.0.0.0")]
```

Comment:
For assemblies like this that we don't ship. Should we just delete the entire `Properties/AssemblyInfo.cs` file?

That is generally what I've don't when porting to SDK-style.

Reply:
I've removed both this and the `AssemblyInfo.cs` file associated with `xa-prep-tasks`.